src/snowflake/snowpark/modin/plugin/_internal/frame.py

```diff
@@ -400,19 +400,25 @@ def data_columns_index(self) -> "pd.Index":
                 convert_to_lazy=False,
             )
 
-    @property
-    def index_columns_pandas_index(self) -> native_pd.Index:
+    def index_columns_pandas_index(
+        self, *, statement_params: Optional[dict[str, str]] = None, **kwargs: Any
```

Comment:
Why is adding `statement_params` here necessary?

Reply:
Oh, I see your point. I copied the arguments from to_pandas but I agree we just need `kwargs` and `statement_params` is not necessary.